utils/create-database.js

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
As in book library, all the comments arent adding any value so could all be removed